src/controllers/trust.details.controller.ts

```diff
@@ -85,7 +85,10 @@ const post = (
       },
     );
 
-    return res.redirect(config.TRUST_DETAILS_URL);
+    logger.debugRequest(req, `Going to the check your answers page (slice 1 only)`);
```

Comment:
The 'slice 1' part in the comment. Is that temporary? Maybe worth adding a TODO comment to indicate and remind you that this will change later on.

Reply:
Updated the comment - please resolve if happy